src/components/Footer/Footer.scss

```diff
@@ -60,7 +60,7 @@
   }
 
   &__link {
-    font-family: 'Mont-Bold'; 
+    font-family: 'Mont-Bold';
```

Comment:
please, add there and same cases second font-family

Reply:
Random change, may be i do it by accident! But in develop branch all good